components/fieldlabel/stories/template.js

```diff
@@ -29,9 +33,20 @@ export const Template = ({
     <label class=${classMap({
       [rootClass]: true,
       [`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
+      [`${rootClass}--${alignment}`]: typeof alignment !== "undefined",
+      "is-disabled": isDisabled,
       ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
-  })}>
-    ${label}
-  </label>
+    })}
+      style="width: 72px;"
```

Comment:
Would a second story that displays it aligned make sense here?
 [Similar to how SWC does](https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/field-label--side-align-start), but I think it could just be "aligned" and leave the control in there to flip it left to right. 

I do see that we are using this width for the left and right aligned on the docs site but not with the standard variant. 

I bring this up because I was locally adjusting the text length and if it is required or not and with the width fixed you get some unexpected text wrapping.